hw/dv/tools/xcelium/common_cov.vRefine

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
+<refinement-file-root>
+  <information comment-version="1" creation-time="Wed 10 Feb 2021 04:11:52 PST" creator="rasmus.madsen" csCheck="false" modified-by="rasmus.madsen" modify-time="Wed 10 Feb 2021 05:18:29 PST" save-ref-method="seq" tool-version="Cadence vManager20.03" rules-signature-c="1ead9ec7f5923387d479a9c435d026b0">
+    <ucm-files>
+      <ucm-file domain="icc" modelCheckSum="1468209578" path="/wdc/proj/cto/top/work/rasmus.madsen/opentitan/golden/hw/ip/aes/dv/scratch/tools_cov/aes-sim-xcelium/cov_merge/merged/icc_578319aa_69124526.ucm"></ucm-file>
```

Comment:
this is meant to be a common cov cfg file, but it contains your local path, not sure if we can remove it
does this work for other IPs?

Reply:
ahh good catch - it is the output of the vRefine when saved through the gui.
not sure if it matters I will look into alternatives